packages/htmlviewer/src/widget.tsx

```diff
@@ -201,9 +201,8 @@ export class HTMLViewer
 
     // Inject dynamic style for links if the document is not trusted
     if (!this.trusted) {
-      const warning = this.translator
-        .load('jupyterlab')
-        .__('Action disabled as the file is not trusted.');
+      const trans = this.translator.load('jupyterlab');
+      const warning = trans.__('Action disabled as the file is not trusted.');
```

Comment:
cc @fcollonval @krassowski to double check if the "Check internationalization changes / check_i18n" CI failure is something to worry about (looks like the actual string did not change so it should be fine?)

Reply:
Now, this is expected here because the string was wrongly not detected in previous version sou would not be translated anyways.